tests/test_dsp_resample.py

```diff
@@ -0,0 +1,94 @@
+import pyfar as pf
+import pytest
+import numpy as np
+
+
+def test_upsampling():
```

Comment:
This and the next test could be combined using
```suggestion
@pytest.mark.parametrize('L', [2, 0.5])
def test_upsampling(L):
```
with L being the resampling factor